test/Makefile

```diff
@@ -127,6 +127,9 @@ ifdef SEPARATE
   CILLY+= --nomerge
 endif
 
+# Disable GCC 14 new warnings as errors because some tests contain them
+CFLAGS += -Wno-error=implicit-int -Wno-error=implicit-function-declaration -Wno-error=int-conversion -Wno-error=incompatible-pointer-types
+
```

Comment:
What happens with old GCC versions? Do they already have these options?

Reply:
Yes, I checked with GCC 7-13 (everything older we still look for in our `RealGccConfigure`) in Godbolt, that these already existed, but just weren't errors by default, so these shouldn't be a problem.
GCC 14 added a few new ones I had to exclude here (in order to avoid needing GCC-version specific configuration logic here).